js/screens/pause/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { StyleSheet, Text, View, TouchableOpacity, Image } from 'react-native';
```

Comment:
This was unused right? We should get some basic linter magic going to prevent this kinda stuff. Will create a ticket.

Reply:
Yes, it was obsolete. Introducing ESLint or something is a good idea though!